packages/codec/README.md

```diff
@@ -1,15 +1,36 @@
-# Truffle Decoder Core
-This module provides interfaces for decoding Solidity variables.  This is a
-fairly low-level interface meant to be used by e.g. a debugger; for a
-higher-level interface, see the `truffle-contract-decoder` package instead.
+# Truffle Codec
 
-## Usage
-```
-import { forEvmState } from 'truffle-decoder-core';
+This module, `@truffle/codec`, provides an interface for decoding Solidity
+smart contract state as well as information sent to or from smart contracts
+using the Solidity ABI.  It produces output in a machine-readable form that
+avoids losing any information.  It also has some rudimentary encoding
+functionality, that will be expanded in the future.  This module is also
+what Truffle Debugger uses for its decoding.
+
+This is a low-level module, and it's probable that you should use the
+higher-level interface [Truffle
+Decoder](https://github.com/trufflesuite/truffle/tree/master/packages/decoder)
```

Comment:
Can this be a relative link instead of `master`?  If it's not easily doable, that's fine, but I would think you could just link to `../decoder/`